Userland/Applications/SoundPlayer/Player.cpp

```diff
@@ -12,7 +12,7 @@ Player::Player(Audio::ConnectionFromClient& audio_client_connection)
     , m_playback_manager(audio_client_connection)
 {
     m_playback_manager.on_update = [&]() {
-        auto samples_played = m_audio_client_connection.get_played_samples();
+        auto samples_played = m_audio_client_connection.total_played_samples();
```

Comment:
oh god no, this is very incorrect as soon as we play another file. we should get this from the loader instead. why is this code here in the first place, wtf?